usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
```

Comment:
When we use this function, I'd expect the data will be in GCS rather than on the local filesystem, so a file path won't be helpful in GCS. I'd have the function take a file / file-like object so it can function in both contexts.

Reply:
The next step after reading header from GeoTIFF will be to cut giant TIFF into smaller chunks using GDAL utility which cannot work with remote file-like blobs on GCS. So I think we would have to load this file to local FS first. WDYT?

Reply:
I'm saying that if the interface is a file-like object, we can handle both use cases.
If we have

```python
def load_elevation_from_geotiff(file: typing.BinaryIO, ...):
```
we can call it with
```python
with open(file_path) as fd:
  load_elevation_from_geotiff(fd)
```
or with
```python
with bucket.blob(gcs_path).open() as fd:
  load_elevation_from_geotiff(fd)
```

Reply:
Once the chunks are created, we'll need to read the tiff chunk in the code preparing the feature matrix and this function could be useful in that context.

Reply:
Done